src/main/resources/org/jenkinsci/plugins/workflow/steps/MailStep/help.html

```diff
@@ -1,3 +1,6 @@
 <div>
     Simple step for sending email.
+
+    <p>Use the <a href="https://www.jenkins.io/redirect/pipeline-snippet-generator">Pipeline Snippet Generator</a> to generate a sample pipeline script for the mail step.</p>
```

Comment:
and so on... 😄

Reply:
okay, thank you😊. Let me do the rest

Reply:
@mRamonLeon, since I am adding the same sentence in the add-code examples PR, I was wondering if this PR is still needed?